src/Shell/QueueShell.php

```diff
@@ -65,6 +67,23 @@ public function initialize() {
 			}
 		}
 
+		//configure the logger if it is set to true
+		if (Configure::read('Queue.log')) {
+			Log::drop('debug');
+			Log::drop('error');
```

Comment:
I usually use https://github.com/dereuromark/cakefest/blob/3.0/config/app_custom.php#L23 etc and scoped logging in that case, but I am also fine with this approach in that case.
It is very isolated in the queue shell after all,.